ask-sdk-core/tst/attributes/AttributesManagerFactory.spec.ts

```diff
@@ -189,7 +228,7 @@ describe('AttributesManagerFactory', () => {
 
     });
 
-    it('should thrown an error when trying to save persistent attributes without persistence manager', async() => {
+    it('should thrown an error when trying to save persistent attributes without persistence adapter', async() => {
```

Comment:
nit: `throw`

Reply:
will fix in the next PR with some other minor fixes